warp10/src/main/java/io/warp10/script/ext/http/HttpWarpScriptExtension.java

```diff
@@ -0,0 +1,172 @@
+//
+//   Copyright 2021  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.ext.http;
+
+import io.warp10.WarpConfig;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.warp.sdk.WarpScriptExtension;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Extension for HTTP and the associated function to change limits: MAXURLCOUNT and MAXDOWNLOADSIZE
```

Comment:
These functions do not need to exist. Rely on capabilities to modify the limits.